### PR TITLE
Add Rumble extractor

### PR DESCRIPTION
**Pull request by shavit**
Support for Rumble https://github.com/iawia002/lux/issues/1188

### CODE REVIEW

File: extractors/rumble/rumble.go [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+		return nil, errors.WithStack(err)
+	}
+
+	html := string(b)
+
+	cookiesArr := make([]string, 0)
+	cookies := res.Cookies()
+
+	for _, c := range cookies {
+		cookiesArr = append(cookiesArr, c.Name+"="+c.Value)
```
Comment (iawia002) [inline extractors/rumble/rumble.go:72]: `cookiesArr` is never used, do we need this?

**Review by iawia002 — COMMENTED** [review]
And you need to update README here: https://github.com/iawia002/lux#supported-sites

File: extractors/rumble/rumble.go (hunk 1 above)
Comment (shavit) [reply extractors/rumble/rumble.go]: Right, I'll make a couple more changes.

**Review by iawia002 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**